### ISSUE TITLE
Suggestion: include example for how to make select work in page-per-thing / as a page heading

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/govuk-design-system#2120]
The design system page for selects does not show how to use it in a page-per-thing context. To do it, you need to refer to a page such as text input to see the right classes..

![Screenshot 2022-03-28 at 16 51 43](https://user-images.githubusercontent.com/2204224/160438396-6eb059bf-7cce-407d-b5bf-7d45ffb85abf.png)

**Comment by mazzabodwin-dit** [comment]
Yup, this would be super useful as I had no idea how to try and make the label a page header with the correct bottom-margin. In my specific case I am using the Alpha autocomplete (as I know selects are often frowned upon) .

**Comment by sulthan-ahmed** [comment]
thanks @edwardhorsford for raising this and adding it to the backlog @joelanman.  We are actually working on updating our Home Office Forms framework for selects so I'd be interested in see the html.  For now I'll go with @edwardhorsford suggestion of 
> [@Sulthan Ahmed [Home Office]](https://ukgovernmentdigital.slack.com/team/U0QS8TRNE) I think the markup would be similar to an input.
